packages/chain-mon/src/wallet-mon/service.ts

```diff
@@ -0,0 +1,198 @@
+import {
+  BaseServiceV2,
+  StandardOptions,
+  Gauge,
+  Counter,
+  validators,
+  waitForProvider,
+} from '@eth-optimism/common-ts'
+import { getChainId, compareAddrs } from '@eth-optimism/core-utils'
+import { Provider } from '@ethersproject/abstract-provider'
+import mainnetConfig from '@eth-optimism/contracts-bedrock/deploy-config/mainnet.json'
+import goerliConfig from '@eth-optimism/contracts-bedrock/deploy-config/goerli.json'
+import l2OutputOracleArtifactsMainnet from '@eth-optimism/contracts-bedrock/deployments/mainnet/L2OutputOracleProxy.json'
+import l2OutputOracleArtifactsGoerli from '@eth-optimism/contracts-bedrock/deployments/goerli/L2OutputOracleProxy.json'
+
+import { version } from '../../package.json'
+
+const networks = {
```

Comment:
Should be able to get this directly from the chain using only an rpc url imo.   Same with sdk it should be able to self bootstrap from an rpc url.  Otherwise it doesn't scale to an arbitrary amount of OP chains

Reply:
Agree, but a new endpoint seems like a pretty big lift. 
If we don't want to put it in the STK, I'm fine with just doing this for now or maybe putting it into core-Utica